corehq/apps/export/det/schema_generator.py

```diff
@@ -62,9 +64,15 @@ class CaseDETSchemaHelper(DefaultDETSchemaHelper):
     def __init__(self, dd_property_types):
         self.dd_property_types = dd_property_types
 
-    @staticmethod
-    def transform_path(input_path):
-        # either return hard-coded lookup or add prefix
+    def get_path(self, input_column):
+        if isinstance(input_column, CaseIndexExportColumn):
+            # this is an obscure but correct reference to the index reference ID
+            # typically "parent", occasionally "host", rarely miscellaneous other things...
+            # https://github.com/dimagi/commcare-hq/pull/29530/files#r613936070
```

Comment:
nit: Why include the URL of this PR? It's easy enough to lookup with git blame.

Reply:
Good question. I find chasing git history after a few commits is quite tiresome (assuming the code gets edited/moved at some point in the future). Do you have a good workflow for that and do you view a downside to including this?

Reply:
I usually use some combination of VS Code and github.com, and usually don't find it too difficult. IDE support could/should be better, should be able to do it all in VS Code, hehe. In really desperate situations git pickaxe (`git log -p (-S|-G)...`) is handy, if slow.

The main argument against it is that it takes up space and makes the code less readable without adding value unless you're specifically interested in the history of that comment.

Like I said, only a small nit, not a big deal.